.gitignore

```diff
@@ -6,3 +6,6 @@ tags/
 .vscode/
 Build/
 Conf/
+node_modules/

```

Comment:
not opposed to adding more things but interested in what workflow you had that brought these files into the UEFI tree.  I generally install nodejs somewhere outside of the edk code tree.  Is this from installing node into edk source tree?

Reply:
Huh - I could have sworn they were put there automatically, but I indeed cannot reproduce it. Discard as user error. :)

Reply:
Oh dear - running certain npm commands without the -g flag creates the packages.json files in the current directory, even if there is no operation to be performed. E.g., ``npm uninstall cspell`` will create the packages.json files whether or not cspell is actually (locally) installed. So this definitely is a user error. Sorry!